src/server/config-dist.json

```diff
@@ -15,5 +15,13 @@
 		"LED_BRIGHTNESS": 255,
 		"LED_INVERT": false,
 		"LED_CHANNEL": 0
+	},
+	"ADS1X15": {
+		"HARDWARE_NAME": "ADS1115",
+		"CONNECTED_CHANNELS": [0,1],
+		"GAINS": [1,1],
+		"R1_VALUES": [22,22],
+		"R2_VALUES": [3.3,3.3],
+		"CORRECTION_FACTORS": [1.00,1.00]
```

Comment:
Should we add the warnings thresholds for all the sensors here too, like the voltage level that the interface warns you the node voltage is low?

Reply:
I would have that in a separate piece of config, so the thresholds arent tied to the hardware. Maybe something like `
warnings: { "sensorlabel1" :{ min_threshold: 18}, "sensorlabel2" :{ min_threshold: 12}}
`
(forgive my json)
But I would be inclined to do that in a separate PR to avoid this one become too monolithic.

Reply:
Ok. I'll leave them out for now till next pull.